dist/components/alert/_alert.scss

```diff
@@ -18,8 +18,8 @@ $alert__success-color: if(variable-exists(success-color), $success-color, green)
 $alert__error-color: if(variable-exists(error-color), $error-color, red) !default;
 
 
-// Alert Root
-// ----------
+// Alert root
```

Comment:
They were changed to the lowercase to not distract from the component name (the same for modifier). Because `Arrange Modifiers: Horizontal Alignment` is less readable comparing to `Arrange modifiers: Horizontal alignment`